pkg/reconciler/openshift/tektonaddon/tektonaddon.go

```diff
@@ -462,28 +442,14 @@ func makeInstallerSet(ta *v1alpha1.TektonAddon, manifest mf.Manifest, prefix, re
 	}
 }
 
-func (r *Reconciler) deleteInstallerSet(ctx context.Context, ta *v1alpha1.TektonAddon, component string) error {
+func (r *Reconciler) deleteInstallerSet(ctx context.Context, component string) error {
 
```

Comment:
same here, instead of the `component string` let us pass label selector as the last param